README.md

```diff
@@ -62,7 +62,7 @@ One solution to this is to temporarily switch to using PostgreSQL and Sidekiq wi
 ```yaml
 development:
   adapter: redis
-  url: <%= ENV.fetch("REDIS_URL") { "redis://localhost:6379/1" } %>
+  url: <%= ENV.fetch("SIDEKIQ_REDIS_URL") { "redis://localhost:6379/1" } %>
```

Comment:
This is for actioncable, not sidekiq.  Are you sure you want to change it?

Reply:
It seems like in our other apps we keep actioncable included even if it's not actively being used by the app. Maybe we should do this same here? I modified the PR to reference`ACTION_CABLE_REDIS` in `cable.yml`